templates/docker-compose.yml

```diff
@@ -112,3 +115,29 @@ services:
 {% endif %}
     restart: "always"
     logging: *default-logging
+
+{% if pictrs_safety is defined and pictrs_safety|bool == true %}
+  pictrs-safety:
+    image: ghcr.io/db0/pictrs-safety:v1.2.2
+    hostname: "pictrs-safety"
+    networks:
```

Comment:
These networks aren't used anymore.

Reply:
Oh, should I remove them, or replace them with something? As I said, I'm not that good with docker

Reply:
You should be able to remove them. They will cause an error as-is.